scripts/genAddrMainnetOverview.js

```diff
@@ -7,7 +7,7 @@ const formatToEtherscanLink = (l) =>
 
 // Funky protocols with different structures
 // Need to construct the markdown a bit differently
-const funkyProtocols = ["curvefi"];
+const funkyProtocols = ["curvefi", "idle"];
```

Comment:
You'll need to write a section here where you call `markdown.push` on the addresses you want to see show up. We should probably refactor this to be a bit better, but this is what we have right now.

Reply:
in fact, I did use `yarn build:docs` to generate `overview_mainnet_address.md`. As my `contracts` object is deeper than one level, I found that the script for "curvefi" also works for "idle". 
That is why I added it to `funkyProtocols`. Try to run `yarn build:docs` yourself 😃

Reply:
I just did, thank you for the explanation!